### PR TITLE
[feature1] First feature was implemented

### PR DESCRIPTION
**Pull request by zutibr**
Following changes was made:

- new-feature.txt was added
- new-folder was created

### CODE REVIEW

File: new-feature.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This file was created in the feature1 branch.
```
Comment (ayoubft) [inline new-feature.txt:1]: It is really good that you decided to add this line of code.

Comment (ayoubft) [inline new-feature.txt:1]: Good idea you decided to add this line of code.

Comment (zutibr) [reply new-feature.txt:1]: Thank you!

**Review by ayoubft — APPROVED** [review]
Good work!

**Pull request merged**